README.md

```diff
@@ -32,7 +32,7 @@ INSTALLED_APPS = [
 
 ### BitGo
 
-To use the BitGo's wallet, it is necessary to import `BitGoIntegration` class from `polaris_bitgo.bitgo.integration` package and pass it as an argument to the `custody` parameter in the Polaris' `register_integrations` function.
+To use the BitGo's wallet, it is necessary to import `BitGoIntegration` class from `polaris_bitgo.bitgo.integration` package and pass it as the `custody` parameter to Polaris' `register_integrations` function.
```

Comment:
Maybe you could also refer the object of the class on the text, instead of referring just the class, if you pass an object/instance as the parameter, and not the class itself.
If that's the case, I'd go with something like "... and pass an object [or "instance"] of it as the `custody` parameter...".